source/GameData.cpp

```diff
@@ -730,6 +731,21 @@ const vector<string> &GameData::Category(const CategoryType type)
 
 
 
+const BunkType &GameData::GetBunkType(const std::string type)
+{
+	auto it = objects.bunkTypes.find(type);
+	return it->second;
+}
```

Comment:
Looks lile usually a getter isnt provided
I'm all for having one though 

Not worries not smth you have to change just a note